week7/cooper_client_react/09_the_controller.md

```diff
@@ -54,8 +70,8 @@ Next, let's add a `create` method with the following code to our new controller
 # app/controllers/api/v1/performance_data_controller.rb
 class Api::V1::PerformanceDataController < ApplicationController
   def create
-    @data = PerformanceData.new(params[:performance_data])
-    if @data.save
+    data = PerformanceData.new(params[:performance_data])
+    if data.save
       render json: { message: 'all good' }
```

Comment:
tell it to send 200 response status but nothing else

`render :nothing => true, :status => 200`
`head :ok`

Reply:
addressed